source/PlayerInfo.h

```diff
@@ -121,8 +121,9 @@ class PlayerInfo {
 	std::map<const std::shared_ptr<Ship>, std::vector<std::string>> FlightCheck() const;
 	// Add a captured ship to your fleet.
 	void AddShip(const std::shared_ptr<Ship> &ship);
-	// Buy or sell a ship.
+	// Buy a ship, receive a gifted ship, or sell a ship.
```

Comment:
Receive a ship given. (a gifted ship is a bit different than a ship that is given to the player).